README.md

```diff
@@ -108,11 +108,63 @@ before starting kube-apiserver. For that you need to perform the following high
 Note: These steps have been verified with [kops](https://github.com/kubernetes/kops) but 
 it should be similar to any other cluster bootstrapping tool.
 
-#### Run aws-encryption-provider as static pod
-You need to have encryption provider running before kube-apiserver, and to do that you can
-use [static pods](https://kubernetes.io/docs/tasks/administer-cluster/static-pod/) functionality. For kops, static pod manifests are available at `/etc/kubernetes/manifests`. You can further use kops file assets functionality to drop 
-the pod spec file in that directory.
+#### Set the `--encryption-provider` flag with kops
+To set the kubernetes `--encryption-provider` flag with kops you must add it to the kops cluster specificition.
+```yaml
+kind: Cluster
+spec:
+  encryptionConfig: true
 
+```
+Also, you need let kops know about the encryption configuration file. To do this you have to create a kops secret with a special `encryptionconfig` flag during cluster creation, in the following order:
+```bash
+kops create -f <path-to-cluster-spec> --state <state-store>
+kops create secret encryptionconfig -f <path-to-encryption-config-file> --state <state-store> --name <cluster-name>
+kops update cluster <cluster-name> --state <state-store> --yes
+```
+#### Permissions
+Ensure master IAM role has permissions to encrypt/decrypt using the kms. You can achieve this
+using additionalIAMPolicies functionality of kops.
+Example:
+```yaml
+kind: Cluster
+spec:
+  additionalPolicies:
+    node: |
```

Comment:
There shouldn't be any node policies for this provider.

Reply:
This is addressed in commit 727d254.